### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Samsomyajit**
This update is requested in reference to the issue #33552 
[Under the heading "Obtain and Import the Library", the link incorrectly refers to the micro_speech.zip package instead of a person_detection example.]
So I updated the "Obtain and Import the Library" section on  the micro_speech.zip [now: person_detection_example.zip].

### CODE REVIEW

**Comment by rthadur** [comment]
please combine #33582 with this PR

**Comment by Samsomyajit** [comment]
in reference to issue #33552 .... Just like the person detection .zip name in the magic-wand documentation the same problem is present.

**Comment by Samsomyajit** [comment]
@rthadur would you please check this out: when I click on the person_detection.zip link after update this pops out:

`
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error nighteye="disabled">
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Details>
No such object: tensorflow-nightly/github/tensorflow/tensorflow/lite/experimental/micro/tools/make/gen/arduino_x86_64/prj/person_detection/person_detection.zip
</Details>
</Error>
`

**Comment by Samsomyajit** [comment]
@rthadur Why close #33582 ? The magic_wand has the same problem as person_detection in README.md

**Comment by Samsomyajit** [comment]
@rthadur Please check the update made to the README.md and tell me if its fine now

**Comment by mihaimaruseac** [comment]
This should include the changes from #33582?

**Review by adisakshya — COMMENTED** [review]
Yes, the link for ```person_detection``` is fixed. Also the```magic-wand``` documentation needs this same fix as #33582 suggests. You can include the changes from #33582 here in #33579.

**Comment by Samsomyajit** [comment]
> Yes, the link for `person_detection` is fixed. Also the`magic-wand` documentation needs this same fix as #33582 suggests. You can include the changes from #33582 here in #33579.

I have added the magic_wand.zip issue just like the person_detection as requested....

File: tensorflow/lite/experimental/micro/examples/magic_wand/README.md [hunk 1]
```diff
@@ -346,6 +346,7 @@ SLOPE:
   *
  * * * * * * * *
 ```
+> Yes, the link for `person_detection` is fixed. Also the`magic-wand` documentation needs this same fix as #33582 suggests. You can include the changes from #33582 here in #33579.
```
Comment (mihaimaruseac) [inline tensorflow/lite/experimental/micro/examples/magic_wand/README.md]: Please remove this line

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Comment by Samsomyajit** [comment]
@mihaimaruseac PLease check the updated README.md the rquested line is removed.

**Review by mihaimaruseac — DISMISSED** [review]

File: tensorflow/lite/experimental/micro/examples/magic_wand/README.md [hunk 2]
```diff
@@ -26,10 +26,17 @@ The sample has been tested with the following devices:
 
 ### Install the Arduino_TensorFlowLite library
 
+
```
Comment (mihaimaruseac) [inline tensorflow/lite/experimental/micro/examples/magic_wand/README.md]: Remove this additional empty line

File: tensorflow/lite/experimental/micro/examples/magic_wand/README.md [hunk 3]
```diff
@@ -26,10 +26,17 @@ The sample has been tested with the following devices:
...
+Download the current nightly build of the library: [magic_wand.zip](https://storage.googleapis.com/tensorflow-nightly/github/tensorflow/tensorflow/lite/experimental/micro/tools/make/gen/arduino_x86_64/prj/magic_wand/magic_wand.zip)
+
+Next, import this zip file into the Arduino Desktop IDE by going to `Sketch ->
+Include Library -> Add .ZIP Library...`.
+
 This example application is included as part of the official TensorFlow Lite
 Arduino library. To install it, open the Arduino library manager in
 `Tools -> Manage Libraries...` and search for `Arduino_TensorFlowLite`.
 
+
```
Comment (mihaimaruseac) [inline tensorflow/lite/experimental/micro/examples/magic_wand/README.md]: Remove this additional empty line

File: tensorflow/lite/experimental/micro/examples/person_detection/README.md [hunk 4]
```diff
@@ -42,10 +42,14 @@ Connect the Arducam pins as follows:
 
 ### Install the Arduino_TensorFlowLite library
 
+
```
Comment (mihaimaruseac) [inline tensorflow/lite/experimental/micro/examples/person_detection/README.md]: Remove this additional empty line

File: tensorflow/lite/experimental/micro/examples/person_detection/README.md [hunk 5]
```diff
@@ -42,10 +42,14 @@ Connect the Arducam pins as follows:
...
+Download the current nightly build of the library: [person_detection.zip](https://storage.googleapis.com/tensorflow-nightly/github/tensorflow/tensorflow/lite/experimental/micro/tools/make/gen/arduino_x86_64/prj/person_detection/tensorflow_lite.zip)
+
 This example application is included as part of the official TensorFlow Lite
 Arduino library. To install it, open the Arduino library manager in
 `Tools -> Manage Libraries...` and search for `Arduino_TensorFlowLite`.
 
+
```
Comment (mihaimaruseac) [inline tensorflow/lite/experimental/micro/examples/person_detection/README.md]: Remove this additional empty line

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/lite/experimental/micro/examples/magic_wand/README.md (hunk 3 above)
Comment (Samsomyajit) [reply tensorflow/lite/experimental/micro/examples/magic_wand/README.md]: Plz check! I made the necessary changes

File: tensorflow/lite/experimental/micro/examples/magic_wand/README.md [hunk 6]
```diff
@@ -26,6 +26,11 @@ The sample has been tested with the following devices:
 
 ### Install the Arduino_TensorFlowLite library
 
+Download the current nightly build of the library: [magic_wand.zip](https://storage.googleapis.com/tensorflow-nightly/github/tensorflow/tensorflow/lite/experimental/micro/tools/make/gen/arduino_x86_64/prj/magic_wand/magic_wand.zip)
+
+Next, import this zip file into the Arduino Desktop IDE by going to `Sketch ->
+Include Library -> Add .ZIP Library...`.
```
Comment (mihaimaruseac) [inline tensorflow/lite/experimental/micro/examples/magic_wand/README.md]: `Sketch -> ..` should all be on a single line.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Comment by Shusuke-O** [comment]
Hello. I'm trying to check a magic wand example. But I cannot download magic_wand.zip on the README.md.

**Pull request merged**